observability-kit-agent/src/test/java/com/vaadin/extension/instrumentation/client/ObjectMapExporterTest.java

```diff
@@ -0,0 +1,250 @@
+package com.vaadin.extension.instrumentation.client;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.vaadin.extension.conf.ConfigurationDefaults;
+import com.vaadin.extension.instrumentation.AbstractInstrumentationTest;
+import com.vaadin.extension.instrumentation.util.OpenTelemetryTestTools;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ObjectMapExporterTest extends AbstractInstrumentationTest {
+    @BeforeAll
+    public static void setup() {
+        ConfigurationDefaults.spanExporter =
+                OpenTelemetryTestTools.getSpanExporter();
+    }
```

Comment:
It would be good to have also an `@AfterAll` method to reset the static field, to avoid potential issues with other tests in the future